tests/check_docs.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+
+root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def main():
+  build_output = os.path.join(root, 'site', 'build', 'text', 'docs', 'tools_reference', 'emcc.txt')
+  docs_file = os.path.join(root, 'docs', 'emcc.txt')
```

Comment:
this is called `check_docs` but looks like it only checks `emcc.txt`, maybe that could be reflected in the name (`check_emcc_help` maybe?) or at least explained in a comment?